### ISSUE TITLE
Cannot Load Model

### ISSUE THREAD

**Issue by GregBreak** [ChrisTerBeke/CuraThingiBrowserPlugin#165]
Some models fail to load on the build plate. The wheel keeps spinning forever and I have to uninstall the extension to continue. I can't figure out which models work

**Comment by ChrisTerBeke** [comment]
Hi. Could you link to a model that gives this issue? Then I can debug it :)

**Comment by ChrisTerBeke** [comment]
It seems that Thingiverse has introduced a breaking change in their API. I'm getting the following error response:

```
API returned with status 400 and body b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>InvalidArgument</Code><Message>Only one auth mechanism allowed; only the X-Amz-Algorithm query parameter, Signature query string parameter or the Authorization header should be specified</Message><ArgumentName>Authorization</ArgumentName><ArgumentValue>Bearer e287976762b7d5766e8d20163456ba72</ArgumentValue><RequestId>BQ047NJMD7S4FHSP</RequestId><HostId>BacOJ6fV5ILfdFIdJRNNZl8muOHYHUqVwThPYDhfx1Ngo/0YoIux11nKsi3K2rOstehrVh5lMv4=</HostId></Error>'
```

I'll have a look and see if I can fix it. Unfortunately this has happened several times in the past, and Thingiverse does not publish changes to their API so I have to guess what happens sometimes.

**Comment by ChrisTerBeke** [comment]
Very likely fixed by #169. Will make a new release and publish to the Marketplace soon.

**Comment by ChrisTerBeke** [comment]
A new release has been submitted to Ultimaker and is awaiting approval before publishing it in the Marketplace.
